modules/sdk-coin-ada/src/lib/transactionBuilder.ts

```diff
@@ -59,6 +63,21 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
     return this;
   }
 
+  fee(f: BigNum): BigNum {
```

Comment:
do we need to set fee as we can calculating fee internally?

Reply:
you're right, deleting now